### PR TITLE
Fixes #600- Proposal on how to handle connectivity loss

### LINKED ISSUE #600: Proposal on how to handle connectivity loss
**Actual Behaviour**

When you fire up the chat.susi.ai and then cut your internet connectivity and ask Susi the web app is put idle. Even after restoring the internet connection it stays as such. Connectivity loss may also happen while a request is being sent to the server.

**Expected Behaviour**

Please sketch options how SUSI web app should handle net connectivity losses to provide for great user experience. Loss of connectivity can easily happen, e.g. when you are on a train ride or when your net connection is not reliable.

**Steps to reproduce it**

1. Open http://chat.susi.ai in a web browser.
2. Cut your Wifi/unplug your net cable.
3. Ask a question.

bildschirmfoto vom 2017-07-24 23-13-29

4, Reestablish internet connection.

**Questions for Options**
1. How could the user be made aware of connectivity loss conveniently? How do other chat apps deal with connectivity loss/Flight mode?
2. Do we want to disable the option to ask questions when there is a connectivity loss?
3. How can we ensure that idle questions get answered once the connectivity is back?
4. Do we want to implement a minimum local/client side skill set for the case of no net connectivity such as: "Sorry, can't answer that now. I have no net connectivity"?

### PR DESCRIPTION
**Pull request by rishiraj824**
Fixes issue #600 

Changes: 
- Added message from SUSI, to remind the user if there is no connectivity or connection gets lost, similar to all clients.
 
To test, please turn off your wifi and send a message. Also, reconnect and then send the message to test if the messages are working again.

Screencast :- 
ezgif com-video-to-gif

Demo Link:
http://susi-internet.surge.sh

### CODE REVIEW

File: src/components/ChatApp/MessageListItem/Feedback.react.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ class Feedback extends React.Component {
 	parseSkill = () => {
 		let message = this.props.message;
 		let rating ={};
+		if(message.response===undefined || message.response=== null){
```
Comment (tabesin) [inline src/components/ChatApp/MessageListItem/Feedback.react.js]: Style: blanks or no blanks.

File: src/actions/API.actions.js [hunk 2]
```diff
@@ -84,7 +93,8 @@ export function createSUSIMessage(createdMessage, currentThreadID, voice) {
     url = url+'&latitude='+_Location.lat+'&longitude='+_Location.lng;
```
Comment (tabesin) [inline src/actions/API.actions.js]: Why not += ?

**Comment by rishiraj824** [comment]
@tabesin the functions `handleOffline` and `handleOnline` are responding to independent eventListeners `online` and `offline`, hence two functions. I have made the changes to the other files. Please check.

**Comment by mariobehling** [comment]
Thank you.

**Pull request merged**